mappings/net/minecraft/world/biome/source/util/VanillaTerrainParameters.mapping

```diff
@@ -29,8 +29,8 @@ CLASS net/minecraft/unmapped/C_vtphaipm net/minecraft/world/biome/source/util/Va
 		ARG 1 noise
 	METHOD m_hsxbmunf peaksAndValleys (F)F
 		ARG 0 weirdness
-	METHOD m_htxzicvn calculateSlope (FFFF)F
-	METHOD m_jgcvxkpg mountainContinentalness (FFF)F
+	METHOD m_htxzicvn calcSlope (FFFF)F
```

Comment:
I dont like `calc`. Would much rather have `calculate`

Reply:
knew it. Will sort that now

Reply:
Fixed